stdlib/public/SDK/Foundation/AffineTransform.swift

```diff
@@ -180,13 +192,13 @@ public struct AffineTransform : ReferenceConvertible, Hashable, CustomStringConv
      the `transformStruct`'s affine transformation matrix.
      The resulting matrix takes the following form:
      
-                 [ m11_T  m12_T  0 ] [ m11_M  m12_M  0 ]
-         T * M = [ m21_T  m22_T  0 ] [ m21_M  m22_M  0 ]
-                 [  tX_T   tY_T  1 ] [  tX_M   tY_M  1 ]
+     [ m11_T  m12_T  0 ] [ m11_M  m12_M  0 ]
```

Comment:
I think we're losing readability on these comments, unfortunately; doesn't affect the code, of course, but this is a regression of sorts